Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -67,6 +68,11 @@ def tests
         FileUtils.rm_f "test/coverage/.resultset.json"
       end
 
+      if args.byebug?
+        require "byebug"
+        ENV["HOMEBREW_DEBUG"] = "1"
```

Comment:
`HOMEBREW_DEBUG` will unfortunately alter the behaviour of tests so probably needs to be unset 😭